calendarsubscription/calendarsubscription.go

```diff
@@ -35,28 +35,26 @@ type iCalOptions struct {
 	Version         string         `json:"v,omitempty"`
 }
 
-var iCalTemplate = template.Must(template.New("ical").Parse(`
-	BEGIN:VCALENDAR
-	PRODID:-//GoAlert//{{.Version}}//EN
-	CALSCALE:GREGORIAN
-	METHOD:PUBLISH
-	{{range .Shifts}}
-	BEGIN:VEVENT
-	SUMMARY:On-Call Shift
-	DTSTART:{{.Start}}
-	DTEND:{{.End}}
-	END:VEVENT
-	{{end}}
-
-	{{range .ReminderMinutes}}
-	BEGIN:VALARM
-	ACTION:DISPLAY
-	DESCRIPTION:REMINDER
-	TRIGGER:-PT{{.}}M
-	END:VALARM
-	{{end}}
-
-	END:VCALENDAR
+// RFC can be found at https://tools.ietf.org/html/rfc5545
+var iCalTemplate = template.Must(template.New("ical").Parse(`BEGIN:VCALENDAR
+PRODID:-//GoAlert//{{.Version}}//EN
+CALSCALE:GREGORIAN
+METHOD:PUBLISH
+{{range .Shifts}}
+BEGIN:VEVENT
+SUMMARY:On-Call Shift
+DTSTART:{{.Start}}
+DTEND:{{.End}}
+{{range .ReminderMinutes}}
```

Comment:
Since we're inside the range of `.Shifts` we won't have access to `.ReminderMinutes` without putting it in a variable first.

```
--- FAIL: TestRenderICalFromShifts (0.00s)
    --- FAIL: TestRenderICalFromShifts/ical (0.00s)
        calendarsubscription_test.go:22: err = render ical template:: template: ical:10:8: executing "ical" at <.ReminderMinutes>: can't evaluate field ReminderMinutes in type oncall.Shift; want nil
```

Something like this:
```go
{{- $mins := .ReminderMinutes }}
{{range .Shifts}}
{{range $mins}}
```